support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/PrintProductsHeading.tsx

```diff
@@ -1,65 +1,39 @@
-import { css } from '@emotion/react';
 import type { ActiveRatePlanKey } from 'helpers/productCatalog';
-
-const yellowHighlightText = css`
-	background-color: #ffe500;
-	padding: 0 5px;
-`;
-
-function YellowHighlightText({ text }: { text: string }) {
-	return <span css={yellowHighlightText}>{text}</span>;
-}
+import YellowHighlightText from './YellowHighlightText';
 
 export default function PrintProductsHeading({
 	isObserverPrint,
 	ratePlanKey,
-	isPending,
 }: {
 	isObserverPrint: boolean;
 	ratePlanKey?: ActiveRatePlanKey;
-	isPending: boolean;
 }) {
 	if (isObserverPrint) {
-		const statusText = isPending ? (
-			<>
-				Your <YellowHighlightText text="Observer subscription" /> is being
-				processed
-			</>
-		) : (
-			<>
-				You are now an <YellowHighlightText text="Observer subscriber" />.
-			</>
-		);
 		return (
 			<>
-				{statusText}
+				You are now an{' '}
+				<YellowHighlightText>Observer subscriber</YellowHighlightText>.
 				<br />
 				Welcome and thank you for supporting Observer journalism!
 			</>
 		);
 	}
 
 	if (!ratePlanKey) {
-		return <></>;
+		return null;
```

Comment:
Is there a preference to use `null` over `<></>`, just interested so I can follow this approach in future?